src/pod.ts

```diff
@@ -1616,15 +1616,13 @@ export interface PodConnectionsAllowFromOptions {
  */
 export class PodConnections {
 
-  constructor(protected readonly instance: AbstractPod) {}
+  constructor(protected readonly instance: AbstractPod) {
+    this.defaultNetworkPolicy();
```

Comment:
We shouldn't isolate by default, only upon user request.

Reply:
Will update this.